packages/jellyfish-api-core/src/category/oracle.ts

```diff
@@ -128,6 +137,12 @@ export interface UTXO {
 export interface OracleTokenPrice {
   token: string
   currency: string
+  /**
+   * @example 0.5
+   */
   amount: number
+  /**
+   * @example 1623161076073
```

Comment:
@monstrobishi is this format correct?

Reply:
Should be divided by 1000

Reply:
@jingyi2811 be detailed about this in JSDoc write out the specification.